test/unit/alignment/configuration/align_config_edit_test.cpp

```diff
@@ -12,8 +12,8 @@
 TEST(align_cfg_edit, is_global)
 {
     using type_of_cfg = decltype(seqan3::get<seqan3::align_cfg::mode>(seqan3::align_cfg::edit).value);
-    EXPECT_EQ((std::is_same_v<std::remove_reference_t<type_of_cfg>,
-                              seqan3::detail::global_alignment_type>),
+    EXPECT_TRUE((std::is_same_v<std::remove_reference_t<type_of_cfg>,
+                                seqan3::detail::global_alignment_type>),
                true);
```

Comment:
You need to remove this `, true` when changing from `_EQ` to `_TRUE`. 
Sorry for not being more specific earlier.

Reply:
This happens when you stop thinking. I corrected the commit.